### PR TITLE
Feature5

### PR DESCRIPTION
**Pull request by egatsak**
Fixed 'delete' button logics (now renders only at the Completed tab)
Fixed 'delete all' button logics (now deletes only completed todos)
Added pictures for 'delete' & 'delete all'
Added footer content
Added transform: translate for the buttons when clicked
minor css edit

### CODE REVIEW

File: src/components/Main/AddTodo/AddTodo.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        completed: false,
+        date: new Date(),
+      };
+      setTodos([...todos, newTodo]);
+      setTodo("");
+      e.target.reset();
+    }
+    e.preventDefault();
+  };
+  return (
```
Comment (Vadimchesh) [inline src/components/Main/AddTodo/AddTodo.jsx:26]: add an empty line above

File: src/components/Main/Todos/Todo/Todo.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        />{" "}
+        <span
+          className={
+            todo.completed
+              ? `${styles.textTodo} ${styles.stroke}`
+              : styles.textTodo
+          }
+        >
+          {" "}
+          {todo.text} была создана {todo.date.toLocaleString()}
```
Comment (Vadimchesh) [inline src/components/Main/Todos/Todo/Todo.jsx:40]: I don't think we really need `{todo.date.toLocaleString()` here.

File: src/components/Main/Todos/Todo/Todo.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        </span>
+      </div>
+      {tabs[2].isActive && (
+        <span>
+          <svg
```
Comment (Vadimchesh) [inline src/components/Main/Todos/Todo/Todo.jsx:45]: You can import `svg` as a ReactComponent

File: src/components/Main/AddTodo/AddTodo.jsx (hunk 1 above)
Comment (egatsak) [reply src/components/Main/AddTodo/AddTodo.jsx:29]: +

File: src/components/Main/Todos/Todo/Todo.jsx (hunk 2 above)
Comment (egatsak) [reply src/components/Main/Todos/Todo/Todo.jsx]: + deleted

File: src/components/Main/Todos/Todo/Todo.jsx (hunk 3 above)
Comment (egatsak) [reply src/components/Main/Todos/Todo/Todo.jsx]: + imported

File: src/components/Footer/footer.module.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.footer {
+  flex: 0 0 auto;
+  margin-bottom: 24px;
+  font-family: "Montserrat";
+  font-style: normal;
+  font-size: 14px;
+  line-height: 17px;
+  text-align: center;
+  color: #a9a9a9;
```
[the remark this answers is not in the document]
Comment (egatsak) [reply src/components/Footer/footer.module.css]: I have tried to - see the last commit

[the remark this answers is not in the document]
Comment (Vadimchesh) [reply src/components/Footer/footer.module.css]: move all variables in index.css then you can use them in all CSS files. also you can create a standalone colors.css file, but it is not necessary for this project

[the remark this answers is not in the document]
Comment (egatsak) [reply src/components/Footer/footer.module.css]: done

File: src/index.css [hunk 5]
```diff
@@ -1,13 +1,376 @@
...
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+ */
+
+:root {
+  --blue-color: #2f80ed;
+  --main-text-color: #a9a9a9;
+  --deleteAll-button-color: #eb5757;
```
Comment (Vadimchesh) [inline src/index.css:17]: rename this variable, something like `red-color` or `sensitive-color`

**Pull request merged**